build.gradle

```diff
@@ -53,7 +53,7 @@ allprojects {
 ext {
     // POM file
     GROUP = "com.dropbox.mobile.store"
-    VERSION_NAME = "4.0.0-SNAPSHOT"
+    VERSION_NAME = "4.0.0-alpha02-SNAPSHOT"
```

Comment:
@yigit should this stay as snapshot or is it now going to be alpha02 snapshot?

Reply:
hmm idk but i think the right thing is to keep it as snapshot so that if someone wants to depend on tip of tree 4.0.0, they don't need to keep changing it.

Reply:
```suggestion
    VERSION_NAME = "4.0.0-SNAPSHOT"
```